datetime_formatter/__openerp__.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+{
+    "name": "Date & Time Formatter",
+    "summary": "Helper functions to give correct format to date[time] fields",
+    "version": "1.0.0",
```

Comment:
Can you use the right version scheme: `8.0.1.0.0`  https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers